src/app/detail/[task_id]/[user_type]/actions.ts

```diff
@@ -45,6 +45,13 @@ export const updateWorkflowStateIdOfTask = async (token: string, taskId: string,
   }
 }
 
+export const clientUpdateTask = async (token: string, taskId: string, targetWorkflowStateId: string) => {
+  await fetch(`${apiUrl}/api/tasks/${taskId}/complete?token=${token}&workflowStateId=${targetWorkflowStateId}`, {
```

Comment:
This works, but intuitively it wouldn't make sense for an endpoint named `/complete` to allow a task to be changed to other types of workflowStates as well. E.g. I could hit `/complete` to change my task from a state of complete to unstarted too.

Let me know your thoughts on this.

Reply:
Im thinking about changing the name of the route from 'complete' to 'client'. does that sound good? Also, at first I thought about having separate methods/route for complete and updatetasks, but since both of these function are so similar and they are only used by clients. I decided to merge them. What do you think?